rust/gui-client/src-tauri/Cargo.toml

```diff
@@ -46,7 +42,7 @@ tauri = { version = "1.6", features = [ "dialog", "shell-open-api", "system-tray
 tauri-runtime = "0.14.2"
 tauri-utils = "1.5.3"
 thiserror = { version = "1.0", default-features = false }
-tokio = { version = "1.36.0", features = ["signal", "time"] }
+tokio = { version = "1.36.0", features = ["signal", "time", "macros", "rt", "rt-multi-thread"] }
```

Comment:
Neat, it added those features that were accidentally pulled in from other packages in the workspace. That's been bugging me.

Reply:
I had to add those manually :)